Source/Widgets/InfoBox/InfoBoxViewModel.js

```diff
@@ -142,23 +161,33 @@ define([
         return (this.maxHeight - offset) + 'px';
     };
 
-    var sanitizerTaskProcessor;
-    function defaultSanitizer(html) {
-        if (!defined(sanitizerTaskProcessor)) {
-            sanitizerTaskProcessor = new TaskProcessor('sanitizeHtml', Infinity);
+    defineProperties(InfoBoxViewModel, {
+        /**
+         * Gets or sets the default HTML preprocessor function to use for all instances.
```

Comment:
Perhaps we can think of a better name than "processor" for what this is.  The previous "sanitizer" suggested its purpose more clearly.  What are the use cases for this function now?  It may not be worthy of special treatment anymore.  Users could instead subscribe to the 'description' observable to intercept and modify the description HTML if they really needed to.

Up to you.